test/TestNestedJinjaRule.py

```diff
@@ -21,40 +21,51 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-import unittest
+import os
+import pytest
+from collections import namedtuple
+from ansiblelint import Runner, RulesCollection
```

Comment:
collections is stdlib, pytest is thirdparty, ansiblelint is local:
```suggestion
from collections import namedtuple

import pytest

from ansiblelint import Runner
```